src/components/m-table-body-row.js

```diff
@@ -10,6 +10,26 @@ import PropTypes from 'prop-types';
 import * as CommonValues from '../utils/common-values';
 import { useDoubleClick } from '../utils/hooks/useDoubleClick';
 
+function DetailPanelContent(props) {
```

Comment:
It looks like I'm missing something here so I will standby til @Domino987 needs some help. I apologize, I thought we were making an MTableDetailPanel component so that people could use it to override.

Ignore me!

Reply:
Hi, I as thiniing about it but it would break some animation and stuff if we allow to override this. They already have the content to be editing. Is that enough?